src/core/drive/form_submission.ts

```diff
@@ -24,6 +24,20 @@ export enum FormSubmissionState {
   stopped,
 }
 
+export enum FormEnctype {
+  applicationXWWWFormUrlEncoded = "application/x-www-form-urlencoded",
+  multipartFormData             = "multipart/form-data",
+  textPlain                     = "text/plain"
```

Comment:
What do you think about shortening these a bit?

```suggestion
  urlEncoded = "application/x-www-form-urlencoded",
  multipart  = "multipart/form-data",
  plain      = "text/plain"
```